src/main/resources/application.yml

```diff
@@ -86,4 +86,9 @@ management:
         time-to-live: 2000ms
 
 oauth:
-  endpoint.url: http://localhost:8090/auth
\ No newline at end of file
+  endpoint.url: http://localhost:8090/auth
+
+azure:
+  application-insights:
+    web:
+      enabled: false
```

Comment:
needed to stop servlet being initialised

Reply:
But doesn't this stop `requests` being sent to App Insights - that's what happened in the lookup service

Reply:
Update: Integration tests won't run without this. We suspect it's because we haven't gone to Spring Boot 3

Reply:
going to open telemetry seems to have done the trick https://github.com/ministryofjustice/hmpps-allocations/pull/332/commits/2e5edc529719631f850d0d798f73a4a7f1265039, would be good to see what is logged in app insights after this goes in